<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkAccessViewer</class>
 <widget class="QWidget" name="NetworkAccessViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetworkAccessViewer</string>
  </property>
  <widget class="QTreeWidget" name="responseDetails">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>281</y>
     <width>256</width>
     <height>217</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="requestDetails">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>281</y>
     <width>341</width>
     <height>217</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>262</y>
     <width>341</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Requests</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="requestList">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>39</y>
     <width>891</width>
     <height>217</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Method</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>URL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Response</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Content Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Info</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>10</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>262</y>
     <width>125</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Response Details</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEditResponseData">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>281</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
